modules/monitoring-configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts_{context}"]
+= Configuring different alert receivers for default platform alerts and user-defined alerts
+
+You can configure different alert receivers for default platform alerts and user-defined alerts to ensure the following results:
+
+* All default platform alerts are sent to a receiver owned by the team in charge of these alerts.
+* All user-defined alerts are sent to another receiver so that the team can focus only on platform alerts.
+
+You can achieve this by using the `openshift_io_alert_source="platform"` label that is added by the Cluster Monitoring Operator to all platform alerts.
+
+.Prerequisites
+
+ifndef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
+endif::openshift-dedicated,openshift-rosa[]
+
```

Comment:
I think we should add one prerequisite like others in doc:
`You have installed the OpenShift CLI (oc).`